backend/resolvers/types/pieceType.js

```diff
@@ -0,0 +1,29 @@
+import { IMAGE_ENTRY, VIDEO_ENTRY, OTHER_ENTRY } from "../../constants";
```

Comment:
This is a template for the Apollo piece type. Gets used by the PhotoPiece, VideoPiece, and OtherMediaPiece.